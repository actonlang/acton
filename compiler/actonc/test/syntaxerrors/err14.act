# Invalid assignment target
def bad_assign():
    1 + 2 = 3