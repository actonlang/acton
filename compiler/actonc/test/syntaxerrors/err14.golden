Building file test/syntaxerrors/err14.act using temporary scratch directory
[error Parse error]: unexpected "= "
                     expecting ';', call arguments, slice/index expression, comma, end of line, if clause, or operator

     +--> test/syntaxerrors/err14.act@3:11-3:12
     |
   3 |     1 + 2 = 3
     :           ^ 
     :           `- unexpected "= "
     :              expecting ';', call arguments, slice/index expression, comma, end of line, if clause, or operator
     :              
-----+
